Tasks/AzureRmWebAppDeployment/postDeploymentScript/deleteLogFile.cmd

```diff
@@ -0,0 +1,22 @@
+@echo off
+
+DEL "mainCmdFile_%1.cmd" 
+
+DEL "kuduPostDeploymentScript_%1.cmd"
+
+:: Do not delete file if the execution is not completed within time range
```

Comment:
post deployment script is provided by user and user knows what content is.
Main file is provided by us and its content has no impact on the deployment.